test/expected/topo.yml

```diff
@@ -0,0 +1,39 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+nodes:
+  - id: dGVzdC11aTxicm93c2VyPg==.1
+    name: test-ui<browser>
+    isReal: true
+  - id: WW91cl9BcHBsaWNhdGlvbk5hbWU=.1
```

Comment:
nit: let’s replace these “magic” string with `b64enc` for better readability and maintainability. See https://github.com/apache/skywalking-infra-e2e/blob/c5d37eb3c06aeb9f0ce03638dbfaf5644ac29df8/test/verify/1.expected.yaml#L37 for example

Reply:
Done